lesson_02/job1_execute.py

```diff
@@ -0,0 +1,13 @@
+from job1.dl.extract import get_extract
+from job1.bl.transform import get_transform_result
+from job1.dl.load import load_raw
+
+def main():
+    period='2022-08-09'
+    page='4'
```

Comment:
this parameter not need to hardcode, if we need to use other pages or date. The one date can have data for several pages